types/cesium/index.d.ts

```diff
@@ -0,0 +1,5114 @@
+// Type definitions for cesium 1.44
+// Project: http://cesiumjs.org
+// Definitions by: Aigars Zeiza <https://github.com/Zuzon>
+//                 Harry Nicholls <https://github.com/hnipps>
+//                 Jared Szechy <https://github.com/szechyjs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare namespace Cesium {
```

Comment:
Since Cesium is available as both a module and a UMD global, can you author it as so:

```ts
export = Cesium;
export as namespace Cesium;

declare namespace Cesium {
    // everything you already declared goes here...
}
```

Reply:
Yes, it is available as a module, but I've yet to get it to work as one.

```ts
import {Cesium} from 'cesium';
```

results in:

```
WARNING in ./node_modules/cesium/index.js
11:35-42 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

ERROR in ./node_modules/requirejs/bin/r.js
Module parse failed: Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type.
| #!/usr/bin/env node
| /**
|  * @license r.js 2.3.5 Copyright jQuery Foundation and other contributors.
```

As a result I originally left the module support out of the typing file as it seemed like a problem with Cesium's use of requirejs. If you know of a way to solve this problem without a change to Cesium itself let me know. Thanks!

Reply:
I manually compiled with `tsc` and the above works. Turns out to be a webpack issue. I pushed an update that adds module support.

Reply:
@DanielRosenwasser above is the problem I have. I can't import Cesium without Webpack breaking.